source/options.css

```diff
@@ -1,189 +1,69 @@
-html {
-	min-width: 550px;
-	max-width: 700px;
-	margin: auto;
-	overflow-x: hidden;
-}
-
 :root {
-	--background-color: #fff;
-	--text-color: #202124;
-	--secondary-text-color: #5f6368;
-	--link-color: #3367d6;
-	--input-border-color: rgba(0, 0, 0, 0.2);
-	--github-green: #28a745;
 	--github-red: #cb2431;
+	min-width: 550px;
 }
 
-@media (prefers-color-scheme: dark) {
-	:root {
-		--background-color: #292a2d;
-		--text-color: #e8eaed;
-		--secondary-text-color: #9aa0a6;
-		--link-color: #8ab4f8;
-		--input-border-color: rgba(255, 255, 255, 0.3);
-	}
-}
-
-body {
-	color: var(--text-color);
-	background-color: var(--background-color);
-}
-
-input,
-textarea,
-button {
-	background-color: var(--background-color);
-	color: var(--text-color);
-	border: 1px solid var(--input-border-color);
-}
-
-input:not([type='checkbox']):invalid {
-	color: var(--github-red) !important;
-	border: 1px solid;
+p {
+	margin-top: 0;
 }
 
-input:not([type='checkbox']),
-textarea {
-	display: block;
-	box-sizing: border-box;
-	width: 100%;
+:root [name='customCSS'],
+:root [name='personalToken'] {
```

Comment:
Could you add a `.monotext` class or something to your package and then we could just mark the inputs in the HTML with such class?